src/issue-data-provider.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+module.exports = class IssueDataProvider {
+
+  /**
+   * @param config
+   * @param {GithubAPI} githubApiClient
+   * @param {Logger} logger
+   */
+  constructor (config, githubApiClient, logger) {
+    this.config = config;
+    this.githubApiClient = githubApiClient;
+    this.logger = logger;
+  }
+
+  /**
+   * @param {int} issueId
+   * @returns {Promise<boolean>}
+   */
+  async isIssueInTheKanban (issueId) {
+    var cardsPromise = this.githubApiClient.projects.getProjectCards(
+      {column_id: this.config.kanbanColumns.toDoColumnId}
+    );
+
+    var allTodoCards = await cardsPromise;
+
+    for (var index = 0; index < allTodoCards.data.length; index++) {
```

Comment:
`allTodoCards.data.forEach` can be used since es5
https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/

Reply:
But can I call a `continue` from inside a `forEach` ?

Reply:
a `return` = `continue`, but my bad, don't see the return true ^^

Reply:
If you want to stop the loop whenever `true` is returned, you can use the native function `some`.
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some